### PR TITLE
Runtime provider reference

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Comment by jwitz** [comment]
@odaneau-astro I only included the most recent release for now, but if you can easily run your script to get the tables for all of the Runtime versions between 9.4.0 and now then I can combine with https://github.com/astronomer/docs/pull/3052 to make a complete reference

**Review by odaneau-astro — APPROVED** [review]
LGTM! Let me get you the values for 9.4 to 11.3

File: astro/runtime-provider-reference.md [hunk 1]
```diff
@@ -0,0 +1,2475 @@
...
+| apache-airflow-providers-mysql         | 5.6.0     |
+| apache-airflow-providers-postgres      | 5.11.0    |
+| apache-airflow-providers-smtp          | 1.7.0     |
+| apache-airflow-providers-sqlite        | 3.8.0     |
+| astronomer-providers-logging           | 1.4.7     |
+
+</TabItem>
+</Tabs>
+
+## Astro Runtime 11.3.0
```
Comment (odaneau-astro) [inline astro/runtime-provider-reference.md:87]: Duplicate

File: astro/runtime-provider-reference.md [hunk 2]
```diff
@@ -0,0 +1,2448 @@
...
+| apache-airflow-providers-mysql         | 5.6.0     |
+| apache-airflow-providers-postgres      | 5.11.0    |
+| apache-airflow-providers-smtp          | 1.7.0     |
+| apache-airflow-providers-sqlite        | 3.8.0     |
+| astronomer-providers-logging           | 1.4.7     |
+
+</TabItem>
+</Tabs>
+
+## Astro Runtime 11.2.0
```
Comment (jedcunningham) [inline astro/runtime-provider-reference.md:87]: We should add the slim versions for 11.0-11.2.

File: astro/runtime-provider-reference.md [hunk 3]
```diff
@@ -0,0 +1,2448 @@
...
+To find the version of a provider package installed in an Astro Runtime image, run:
+
+```sh
+docker run --rm <runtime-image> pip freeze | grep <provider>
+```
+
+For example, to find the version of Celery, run the following command:
+
+```sh
+docker run --rm quay.io/astronomer/astro-runtime:11.0.0 | grep apache-airflow-providers-celery
```
Comment (jedcunningham) [inline astro/runtime-provider-reference.md:24]:
```suggestion
docker run --rm quay.io/astronomer/astro-runtime:11.0.0 pip freeze | grep apache-airflow-providers-celery
```

File: docs-partials/runtime-image-architecture.mdx [hunk 4]
```diff
@@ -92,19 +92,7 @@ Astro Runtime {{RUNTIME_VER}} includes the following pre-installed open source p
...
-```
-docker run --rm <runtime-image> pip freeze | grep <provider>
-```
-
-For example, to find the version of the current package for Astronomer providers, run the following command:
-
-```
-astro dev bash pip freeze | grep astronomer-providers
-```
+Astronomer users specific provider package versions in each release of Astro Runtime. See [Astro Runtime provider package reference] for a list of all providers installed in each release of Astro Runtime.
```
Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:95]:
```suggestion
Astronomer users specific provider package versions in each release of Astro Runtime. See [Astro Runtime provider package reference](runtime-provider-reference.md) for a list of all providers installed in each release of Astro Runtime.
```

**Review by lzdanski — COMMENTED** [review]
Thoughts about adding the slim versions for the 11+ versions?

File: astro/runtime-provider-reference.md (hunk 2 above)
Comment (odaneau-astro) [reply astro/runtime-provider-reference.md:87]: I've send a message to all of you in Slack with the latest file.

File: astro/runtime-provider-reference.md [hunk 5]
```diff
@@ -0,0 +1,2448 @@
...
+| apache-airflow-providers-microsoft-azure | 9.0.1     |
+| apache-airflow-providers-mysql           | 5.5.4     |
+| apache-airflow-providers-openlineage     | 1.7.0     |
+| apache-airflow-providers-postgres        | 5.10.2    |
+| apache-airflow-providers-redis           | 3.6.1     |
+| apache-airflow-providers-smtp            | 1.6.1     |
+| apache-airflow-providers-sqlite          | 3.7.1     |
+| astro-sdk-python                         | 1.8.0     |
+| astronomer-providers                     | 1.19.0    |
+| astronomer-providers-logging             | 1.4.7     |
```
Comment (lzdanski) [inline astro/runtime-provider-reference.md:112]:
```suggestion
<Tabs
    groupid="astro-runtime-1120"
    defaultValue="standard"
    values={[
        {label: 'Standard', value: 'standard'},
        {label: 'Slim', value: 'slim'},
    ]}>
<TabItem value="standard">
| Package Name                             | Version   |
|:-----------------------------------------|:----------|
| apache-airflow-providers-amazon          | 8.20.0    |
| apache-airflow-providers-celery          | 3.6.2     |
| apache-airflow-providers-cncf-kubernetes | 8.0.1     |
| apache-airflow-providers-common-io       | 1.3.1     |
| apache-airflow-providers-common-sql      | 1.12.0    |
| apache-airflow-providers-datadog         | 3.5.1     |
| apache-airflow-providers-elasticsearch   | 5.3.4     |
| apache-airflow-providers-fab             | 1.0.4     |
| apache-airflow-providers-ftp             | 3.8.0     |
| apache-airflow-providers-google          | 10.17.0   |
| apache-airflow-providers-http            | 4.10.1    |
| apache-airflow-providers-imap            | 3.5.0     |
| apache-airflow-providers-microsoft-azure | 9.0.1     |
| apache-airflow-providers-mysql           | 5.5.4     |
| apache-airflow-providers-openlineage     | 1.7.0     |
| apache-airflow-providers-postgres        | 5.10.2    |
| apache-airflow-providers-redis           | 3.6.1     |
| apache-airflow-providers-smtp            | 1.6.1     |
| apache-airflow-providers-sqlite          | 3.7.1     |
| astro-sdk-python                         | 1.8.0     |
| astronomer-providers                     | 1.19.0    |
| astronomer-providers-logging             | 1.4.7     |
</TabItem>
<TabItem value="slim">
| Package Name                           | Version   |
|:---------------------------------------|:----------|
| apache-airflow-providers-celery        | 3.6.2     |
| apache-airflow-providers-common-io     | 1.3.1     |
| apache-airflow-providers-common-sql    | 1.12.0    |
| apache-airflow-providers-elasticsearch | 5.3.4     |
| apache-airflow-providers-fab           | 1.0.4     |
| apache-airflow-providers-ftp           | 3.8.0     |
| apache-airflow-providers-http          | 4.10.1    |
| apache-airflow-providers-imap          | 3.5.0     |
| apache-airflow-providers-mysql         | 5.5.4     |
| apache-airflow-providers-postgres      | 5.10.2    |
| apache-airflow-providers-smtp          | 1.6.1     |
| apache-airflow-providers-sqlite        | 3.7.1     |
| astronomer-providers-logging           | 1.4.7     |
</TabItem>
</Tabs>
```

File: astro/runtime-provider-reference.md [hunk 6]
```diff
@@ -0,0 +1,2448 @@
...
+| apache-airflow-providers-imap            | 3.5.0     |
+| apache-airflow-providers-microsoft-azure | 9.0.1     |
+| apache-airflow-providers-openlineage     | 1.7.0     |
+| apache-airflow-providers-postgres        | 5.10.2    |
+| apache-airflow-providers-redis           | 3.6.1     |
+| apache-airflow-providers-smtp            | 1.6.1     |
+| apache-airflow-providers-sqlite          | 3.7.1     |
+| astro-sdk-python                         | 1.8.0     |
+| astronomer-providers                     | 1.19.0    |
+| astronomer-providers-logging             | 1.4.7     |
```
Comment (lzdanski) [inline astro/runtime-provider-reference.md:138]:
```suggestion
<Tabs
    groupid="astro-runtime-1110"
    defaultValue="standard"
    values={[
        {label: 'Standard', value: 'standard'},
        {label: 'Slim', value: 'slim'},
    ]}>
<TabItem value="standard">
| Package Name                             | Version   |
|:-----------------------------------------|:----------|
| apache-airflow-providers-amazon          | 8.20.0    |
| apache-airflow-providers-celery          | 3.6.2     |
| apache-airflow-providers-cncf-kubernetes | 8.0.1     |
| apache-airflow-providers-common-io       | 1.3.1     |
| apache-airflow-providers-common-sql      | 1.12.0    |
| apache-airflow-providers-datadog         | 3.5.1     |
| apache-airflow-providers-elasticsearch   | 5.3.4     |
| apache-airflow-providers-fab             | 1.0.3     |
| apache-airflow-providers-ftp             | 3.8.0     |
| apache-airflow-providers-google          | 10.17.0   |
| apache-airflow-providers-http            | 4.10.1    |
| apache-airflow-providers-imap            | 3.5.0     |
| apache-airflow-providers-microsoft-azure | 9.0.1     |
| apache-airflow-providers-openlineage     | 1.7.0     |
| apache-airflow-providers-postgres        | 5.10.2    |
| apache-airflow-providers-redis           | 3.6.1     |
| apache-airflow-providers-smtp            | 1.6.1     |
| apache-airflow-providers-sqlite          | 3.7.1     |
| astro-sdk-python                         | 1.8.0     |
| astronomer-providers                     | 1.19.0    |
| astronomer-providers-logging             | 1.4.7     |
</TabItem>
<TabItem value="slim">
| Package Name                           | Version   |
|:---------------------------------------|:----------|
| apache-airflow-providers-celery        | 3.6.2     |
| apache-airflow-providers-common-io     | 1.3.1     |
| apache-airflow-providers-common-sql    | 1.12.0    |
| apache-airflow-providers-elasticsearch | 5.3.4     |
| apache-airflow-providers-fab           | 1.0.3     |
| apache-airflow-providers-ftp           | 3.8.0     |
| apache-airflow-providers-http          | 4.10.1    |
| apache-airflow-providers-imap          | 3.5.0     |
| apache-airflow-providers-postgres      | 5.10.2    |
| apache-airflow-providers-smtp          | 1.6.1     |
| apache-airflow-providers-sqlite        | 3.7.1     |
| astronomer-providers-logging           | 1.4.7     |
</TabItem>
</Tabs>

```

**Review by lzdanski — APPROVED** [review]
Just to confirm, we need to update this with every runtime release?

**Comment by jwitz** [comment]
@lzdanski Yes, details of how to do this have been added to the Runtime release process Notion page

**Pull request merged**